File: sort/sort.js
function bubleSort(arr){
    length = arr.length
    for (let i = 0; i < length; i++) {
        for (let j = 0; j < length - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
            }
        }
    }
    return arr
}
var test_arr = [];
for (let i = 0; i < 10000; i++) {
    test_arr[i] = Math.floor(Math.random() * 10000)
}
console.time('bubbleSort')
bubleSort(test_arr)
console.timeEnd('bubbleSort')

File: sort/sort.java
public class sort{
    public static int[] bubleSort(int[] arr){
        int length = arr.length;
        int temp = 0;
        for (int i = 0; i < length; i++) {
            for (int j = 0; j < length - i - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }
        return arr;
    }
    public static void main(String[] args) {
        int[] arr = new int[10000];
        for (int i = 0; i < 10000; i++) {
            arr[i] = (int)(Math.random() * 1000);
        }
        long startTime = System.nanoTime();
        bubleSort(arr);
        long endTime = System.nanoTime();
        System.out.println("time: " + (endTime - startTime)/1e+6 + " ms");
    }
}
